src/main/java/com/github/kiulian/downloader/base64/JdkBase64Encoder.java

```diff
@@ -0,0 +1,15 @@
+package com.github.kiulian.downloader.base64;
+
+import java.util.Base64;
+
+public class JdkBase64Encoder implements Base64Encoder {
```

Comment:
why do we need this class?

Reply:
To provide a default implementation.
Otherwise, you **must** call `Base64Encoder.setInstance(...)` once, before any search attempt.
This method allows to use a better implementation than mine.

Reply:
For super convenience :)
I'll remove it.